www/gatsby-node.js

```diff
@@ -366,7 +355,6 @@ exports.createPages = ({ graphql, actions, reporter }) => {
             context: {
               slug: node.fields.slug,
               locale,
-              ...getPrevAndNext(node.fields.slug),
```

Comment:
Is this correct to remove this from context here or accident?

Reply:
On purpose. If you look at use-prev-and-next, we change it to rely on `useItemList` instead of calculating it in gatsby-node.